Installer/update-vs.ps1

```diff
@@ -0,0 +1,99 @@
+$installer_name = 'git-credential-manager.exe'
+$title = 'Git Credential Manager for Windows upater for Visual Studio v1.0'
+
+function check_access {
+  # Ensure that we're running elevated, and if we're not then restart as elevated.
+  $current_identity = [System.Security.Principal.WindowsIdentity]::GetCurrent()
+  $current_principle = new-object System.Security.Principal.WindowsPrincipal($current_identity)
+  $administrator_role = [System.Security.Principal.WindowsBuiltInRole]::Administrator
+  
+  if (-not ($current_principle.IsInRole($administrator_role))) {
+    write-host -ForegroundColor Red 'fatal: Access Denied'
+    write-host -ForegroundColor Yellow 'Administrator privilages are required, please run again with administrator privilages.'
```

Comment:
I was really trying to avoid the incredibly annoying (to me at least) output from PowerShell's `write-error` utility. Six-plus lines of output to tell somebody the script stopped abruptly seems like overkill - especially when most of the output is irrelevant.

Example...
```
Administrator privileges are required. Please re-start PowerShell as administrator and re-run this script
At line:1 char:1
+ throw 'Administrator privileges are required. Please re-start PowerSh ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Administrator p...run this script:String) [], RuntimeException
    + FullyQualifiedErrorId : Administrator privileges are required. Please re-start PowerShell as administrator and re-run this script

```
... or ...
```
write-error 'Administrator privileges are required. Please re-start PowerShell as administrator and re-run this script' : Administrator privileges are required. Please re-start PowerShell as administrator and re-run
this script
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException

```
... when this is everything the user needs (and it shows up in color!)
```
Administrator privileges are required. Please re-start PowerShell as administrator and re-run this script
```